packages/serverless/src/domain/use-case/group.ts

```diff
@@ -149,23 +154,33 @@ export class Group {
   /**
    * Updates the metadata of a group based on the provided parameters.
    *
-   * @param params - The parameters specifying the group and the metadata to update, following the `TGroupParamUpdateMetadata` type.
+   * @param paramUpdateMetadata - The parameters specifying the group and the metadata to update, following the `TGroupParamUpdateMetadata` type.
    * @param session - (Optional) The MongoDB session for transactional queries.
    * @returns A promise resolving to `true` if the metadata was updated successfully, otherwise `false`.
    */
   public static async updateMetadata(
-    params: TGroupParamUpdateMetadata,
+    paramUpdateMetadata: TGroupParamUpdateMetadata,
     session?: ClientSession
   ): Promise<boolean> {
-    const { databaseName, groupName, createdBy, newGroupName, newDescription } =
-      params;
+    const {
+      databaseName,
+      groupName,
+      createdBy,
+      newGroupName,
+      newGroupDescription,
+    } = paramUpdateMetadata;
     // Check if user don't input both newDescription & newGroupName
     // Using Falsy to ensure user don't input thing like '', 0, null
-    if (!newGroupName && !newDescription) {
+    if (!newGroupName && !newGroupDescription) {
       throw new Error('Need to provide at least one field to update');
```

Comment:
It would be better if you put this in `Joi` as well, like two layers of verification.